<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QPanelCmdWidget</class>
 <widget class="QWidget" name="QPanelCmdWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>130</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>130</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>130</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>19</y>
     <width>286</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>286</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>286</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="title">
    <string>指令选择</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QToolButton" name="toolButton_mode">
      <property name="text">
       <string>模式</string>
      </property>
      <property name="popupMode">
       <enum>QToolButton::InstantPopup</enum>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="toolButton_flycmd">
      <property name="text">
       <string>遥控</string>
      </property>
      <property name="popupMode">
       <enum>QToolButton::InstantPopup</enum>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="toolButton_flymix">
      <property name="text">
       <string>遥调</string>
      </property>
      <property name="popupMode">
       <enum>QToolButton::InstantPopup</enum>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="toolButton_device">
      <property name="text">
       <string>设备</string>
      </property>
      <property name="popupMode">
       <enum>QToolButton::InstantPopup</enum>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="toolButton_4">
      <property name="text">
       <string>任务</string>
      </property>
      <property name="popupMode">
       <enum>QToolButton::InstantPopup</enum>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>81</y>
     <width>291</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QCheckBox" name="checkBox_c">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>30</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="text">
       <string>请选择需要发送的指令</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="SendButton">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>发送</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
